shell/common/gin_converters/extension_converter.cc

```diff
@@ -16,7 +20,13 @@ v8::Local<v8::Value> Converter<const extensions::Extension*>::ToV8(
   auto dict = gin::Dictionary::CreateEmpty(isolate);
   dict.Set("id", extension->id());
   dict.Set("name", extension->name());
+  dict.Set("path", extension->path());
+  dict.Set("url", extension->url());
   dict.Set("version", extension->VersionString());
+
+  auto manifest = extension->manifest()->value()->Clone();
```

Comment:
There is no need for clone, converting a `base::Value` to V8 already implies cloning.

Reply:
I ended up needing to write another converter template for this. c38dce3

Reply:
I think you can just do `dict.Set("manifest", *(extension->manifest()->value()));`?

Reply:
Looking more closely at `converter.ToV8Value`, it seems like this performs close to the same operation as `Clone()`. Please let me know which you prefer.

Reply:
> 
> 
> I think you can just do `dict.Set("manifest", *(extension->manifest()->value()));`?

Ah, yes. Fixed in f7a4f5d